test/Test/Toml/Gen.hs

```diff
@@ -207,6 +214,7 @@ noneArrayList =
     [ AnyValue . Bool    <$> genBool
     , AnyValue . Integer <$> genInteger
     , AnyValue . Double  <$> genDouble
+    , AnyValue . Double  <$> genDoubleText
```

Comment:
This shouldn't be separate generator in this list. You should combine it with `genDouble`. And you should use `frequency` generator to adjust frequency of value appearance. Let's have:

5 for Inf
5 for -Inf
5 for NaN
50 for Ordinary Double